typescript/commands/error_info.py

```diff
@@ -21,7 +21,11 @@ def run(self, text):
             if region.contains(pt):
                 error_text = text
                 break
+
         if len(error_text) > 0:
-            self.view.set_status("typescript_error", error_text)
-        else:
-            self.view.erase_status("typescript_error")
\ No newline at end of file
+            if PHANTOM_SUPPORT:
+                template = '<body><style>div.error {{ background-color: brown; padding: 5px; }}</style><div class="error">{0}</div></body>'
+                display_text = template.format(error_text)
+                self.view.add_phantom("typescript_error", self.view.sel()[0], display_text, sublime.LAYOUT_BLOCK)
```

Comment:
Interesting I don't see `add_phantom` on the `view` class in the docs (https://www.sublimetext.com/docs/3/api_reference.html#sublime.View). Maybe an oversight. They didn't make changes to this API after introducing it right?